### PR TITLE
Replace Rescaling with Normalization in guides/transfer_learning

### LINKED ISSUE #20: [Guides] Pre-processing of images for Xception model
As a part of the Transfer Learning tutorial on the website, Xception model have been used.
https://github.com/keras-team/keras-io/blob/2c5eb0586493d4602c324d48571814e183ad6e32/guides/transfer_learning.py#L426-L432

https://github.com/keras-team/keras-io/blob/2c5eb0586493d4602c324d48571814e183ad6e32/guides/transfer_learning.py#L530-L532

The preprocessing used to feed in data using the tf.data API as well as with the new Rescaling layer normalizes raw input pixels (0-255) into a range (0, 1). But, as per `keras_applications.xception.preprocess_input`, 
(https://github.com/tensorflow/tensorflow/blob/476ec938b253a9479de09aab88dceec6f0a304ed/tensorflow/python/keras/applications/xception.py#L318-L320) the corresponding preprocess_input uses `mode='tf'` (https://github.com/tensorflow/tensorflow/blob/476ec938b253a9479de09aab88dceec6f0a304ed/tensorflow/python/keras/applications/imagenet_utils.py#L181-L184) which normalizes input pixels in the range (-1, 1) instead of (0, 1).

Sometimes calculating activations (for deep feature extraction step) from pre-trained weights are prone to these type of problems with different preprocessing input ranges. Is the example affected?

### PR DESCRIPTION
**Pull request by swghosh**
Fixes #20. 
Please review this PR. I'll run the autogen script after that.

Thanks.

### CODE REVIEW

**Comment by swghosh** [comment]
I've added the necessary md, ipynb files for this tutobook using the supplied autogen script. Request you to merge this PR. 

Thanks.

**Comment by fchollet** [comment]
I see you included a file that's not supposed to be in there, `scripts/tmp_9753115/transfer_learning.ipynb`. Please remove.

**Comment by swghosh** [comment]
I've made the changes. Thanks.
